config/crd/bases/oran.openshift.io_clusterrequests.yaml

```diff
@@ -104,6 +104,11 @@ spec:
                   name:
                     description: Contains the name of the created ClusterInstance.
                     type: string
+                  nonCompliantAt:
+                    description: Holds the first timestamp when the configuration
+                      was found NonCompliant for the cluster.
+                    format: date-time
+                    type: string
```

Comment:
It doesn't feel like this should be part of the clusterInstanceRef - should we rename that to be more general or create a new property for the policy related items?

Reply:
I'll rename it to `ClusterDetails` since it's more general.